tr2:
  hardware_interface:
     loop_hz: 5 # hz
     joints:
       - a0
       - a1
       - a2
       - a3
       - a4
       - g0
       - h0
       - h1
